person.rb

```diff
@@ -1,10 +1,7 @@
 # FrozenStringLiteral: true
-require_relative 'nameable'
-require_relative 'capitalize_decorator'
-require_relative 'trimmer_decorator'
-
-class Person < Nameable
-  attr_accessor :name, :age, :rentals
+# person.rb
+class Person
+  attr_accessor :name, :age, :rentals, :classroom
```

Comment:
- [ ] Well done initializing @rentals however, A method for adding rentals should be present. [Link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) 
- [ ]     The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association

  ```
def add_rental(book, date)
   
 Rental.new(date, book, self)
  end
```